app/forms.py

```diff
@@ -225,12 +225,6 @@ def clean_mobile_number(self):
         if not mobile_number:
             return
 
-        if mobile_number.startswith('1'):
-            RegexValidator('^\d{11}$', message=_("Enter a valid mobile number in "
-               "the form 14034228916"))(mobile_number)
-        else:
-            RegexValidator('^\d{10}$', message=_("Enter a valid mobile number in "
-                   "the form 14034228916"))(mobile_number)
```

Comment:
Excellent questions, we're using Django 1.4.5 (unfortunately) for this site. That still shipped with `django.contrib.localflavor`, later versions split that out and the source code for it now [lives here]( https://github.com/django/django-localflavor/blob/master/localflavor/ca/forms.py#L48).

[Line 51](https://github.com/django/django-localflavor/blob/master/localflavor/ca/forms.py#L51) shows the validation error and it uses the `_` function to enable it for internationalisation. This means that if french is selected it'll display Django's own french translation of that bit of text.

The reason I removed it from the form (in the `clean_mobile_number` function) is because the form field itself provides the validation we need, we don't need to do this again in the `clean_mobile_number()` function.